users/validators.py

```diff
@@ -5,6 +5,7 @@
 from django.core.exceptions import ValidationError
 from django.utils.deconstruct import deconstructible
 
+from users.common.constants import VALID_EMAIL_DOMAIN_LIST
```

Comment:
please move it do `settings`

Reply:
I moved it, however not sure if I import it in good way.